orchestration/OrchestrationService/ConfigurationBuilder.ps1

```diff
@@ -2,62 +2,62 @@ $rootPath = Split-Path -Path $MyInvocation.MyCommand.Definition -Parent
 $helperPath = Join-Path (Join-Path (Join-Path $rootPath -ChildPath '..') -ChildPath 'Common') -ChildPath 'Helper.psm1';
```

Comment:
Make sure to use -AdditionalPath array here as well.

Reply:
Updated.